sdk/maps/Azure.Maps.Search/tests/.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.0",
```

Comment:
nit:  Was adding the `.vscode` folder intentional?   It's not an asset that we typically include.